tms_objects2.erb

```diff
@@ -267,9 +289,13 @@ FileUtils.mkdir('records')
 #ITERATING THROUGH OBJECTS
 #objects_main_query = tmsclient.execute("SELECT top 10 * FROM [Coboat_ObjectsMainQuery]").each
 objects_main_query = tmsclient.execute("SELECT * FROM [Coboat_ObjectsMainQuery] where ObjectID in (22,6236,515)").each
+#objects_main_query = tmsclient.execute("SELECT * FROM [Coboat_ObjectsMainQuery] where ObjectID in (61579,755,4003)").each
 #22 - main test object
 #6236 - for displayStateEditionWrap
 #515 - for qualifierMeasurements
+#61579 - for EventWrap (ConAltNames)
```

Comment:
For each of these, be as explicit as possible here, so you know what to look for.  Like:
#61579 - nameActorSet/appellationValue - logic: preferred, no display date, no prefix